pkg/util/utils.go

```diff
@@ -682,18 +682,15 @@ func DefaultContainerConfig() *config.Config {
 	return containerConfig
 }
 
-func CreateCidFile(cidfile string, id string) error {
-	cidFile, err := OpenExclusiveFile(cidfile)
+func CreateIDFile(path string, id string) error {
+	cidFile, err := os.Create(path)
```

Comment:
Nice catch, @nalind. 

Since this needs to be resubmitted now, small suggestion: change `cidFile` to `idFile`, and (more importantly) `s/cid/id/` in the error messages on lines 688 and 692. A user seeing `cidfile` when the error triggers on a pod id might otherwise be led astray.

(Also, I don't know how to check if `err` includes `path` or not. If it doesn't, please include `path` in the error messages).

Reply:
The standard library [doc](https://pkg.go.dev/os#Create) for `os.Create()` notes that it returns an `os.PathError` on error, which includes a path name as one of its fields, so no worries on that part.

Reply:
Perfect, thank you, and thanks for educating me on how to find out!